dashboard/app/assets/stylesheets/application.scss

```diff
@@ -829,6 +829,59 @@ $small-footer-standard-width: 400px;
   border-top: initial;
 }
 
+/* Custom styling for the locale & copyright buttons and the copyright flyout,
+ * for Music Lab.
+ */
+.music-black {
```

Comment:
Is there a more generic CSS class we could use here? I feel like this dark mode footer will be very useful (for example, this would be great to have on Javalab)